packages/components/src/unit-control/unit-select-control.tsx

```diff
@@ -55,8 +58,8 @@ export default function UnitSelectControl( {
 		<UnitSelect
 			className={ classes }
 			onChange={ handleOnChange }
-			size={ size }
-			tabIndex={ isTabbable ? null : '-1' }
+			selectSize={ size }
+			tabIndex={ isTabbable ? undefined : -1 }
```

Comment:
TS complained that `null` wasn't a valid value here, but `undefined` is.

Reply:
Makes sense! `tabIndex` is mirroring the HTML attribute and therefore it accepts only `number` or `undefined` (which is equivalent to no values being passed)